src/Type/Schema.php

```diff
@@ -335,20 +335,22 @@ private function loadType(string $typeName) : ?Type
     {
         $typeLoader = $this->config->typeLoader;
 
-        if (! $typeLoader) {
+        if (! isset($typeLoader)) {
             return $this->defaultTypeLoader($typeName);
         }
 
         $type = $typeLoader($typeName);
 
         if (! $type instanceof Type) {
-            throw new InvariantViolation(
-                sprintf(
-                    'Type loader is expected to return a callable or valid type "%s", but it returned %s',
-                    $typeName,
-                    Utils::printSafe($type)
-                )
-            );
+            if (is_callable($type)) {
```

Comment:
Mind adding a comment that we do this for the sake of performance? Otherwise, we can erase this in some refactoring in the future (did this in the past).

Reply:
Sure, no prob. Will touch it up after work.

Reply:
Here there be dragons: a16c80b